content/docs/en/getting-started/2-playground-tutorial.md

```diff
@@ -140,7 +140,7 @@ Use the `<TabView>` component to create a two-tab app.
 1. Change the title of the `<ActionBar>` to reflect the app purpose.
 1. Remove the default `<ScrollView>` block and all its contents that come with the template.<br/>`<ScrollView>` components are top-level layout containers for scrollable content.
 1. Drag and drop the `<TabView>` component in its place.<br/>The Playground applies some code formatting, including taking care of indentation. However, the formatting is applied after the insertion and using the browser's undo feature only reverts the formatting and not the insertion of code.
-1. Configure the height of the `<TabView>` to fill the screen (set it to 100%).
+1. Configure the height of the `<TabView>` to fill the screen (set it to 100%). If you are using Android then you should also add the `androidTabsPosition` property to your `<TabView>` and set it to "bottom", otherwise the tabs will appear on the top of the screen.
```

Comment:
```suggestion
1. Configure the height of the `<TabView>` to fill the screen (set it to 100%).<br/>For Android devices, consider adding the `androidTabsPosition` property and setting it to "bottom". Otherwise, the tabs will appear at the top of the screen.
```